test/run-k8s-integration.sh

```diff
@@ -44,7 +44,7 @@ else
   base_cmd="${base_cmd} --gce-region=${gce_region}"
 fi
 
-if [[ "$overlay_name" =~ .*"gke-release-staging".* ]]; then
+if [[ "$overlay_name" =~ .*"gke-release-staging".* ]] || [[ "$overlay_name" = "stable" ]]; then
```

Comment:
We can remove this check entirely since snapshotter should be in all overlays now

Reply:
This is assuming that no integration tests are run against alpha?